BPRBE.Tests/DependencyServiceTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿using BPRBE.Models.Persistence;
+using BPRBE.Persistence;
+using BPRBE.Services;
+using NSubstitute.ReturnsExtensions;
+
+namespace BPRBE.Tests;
+
+[TestFixture]
+internal class DependencyServiceTests
+{
+    private IDependencyService uut;
```

Comment:
I suggest you merge with my branch because the service now uses IValidatorService as well